Hangfire.MySql/MySqlStorage.cs

```diff
@@ -26,30 +23,26 @@ public class MySqlStorage : JobStorage, IDisposable
 
         public virtual PersistentJobQueueProviderCollection QueueProviders { get; private set; }
 
-        public MySqlStorage(string nameOrConnectionString)
-            : this(nameOrConnectionString, new MySqlStorageOptions())
+        public MySqlStorage(string connectionString)
+            : this(connectionString, new MySqlStorageOptions())
         {
         }
 
-        public MySqlStorage(string nameOrConnectionString, MySqlStorageOptions options)
+        public MySqlStorage(string connectionString, MySqlStorageOptions options)
         {
-            if (nameOrConnectionString == null) throw new ArgumentNullException("nameOrConnectionString");
+            if (connectionString == null) throw new ArgumentNullException("connectionString");
             if (options == null) throw new ArgumentNullException("options");
 
-            if (IsConnectionString(nameOrConnectionString))
+            if (IsConnectionString(connectionString))
             {
-                _connectionString = nameOrConnectionString;
-            }
-            else if (IsConnectionStringInConfiguration(nameOrConnectionString))
-            {
-                _connectionString = ConfigurationManager.ConnectionStrings[nameOrConnectionString].ConnectionString;

```

Comment:
I would suggest leaving the option to provide name of the connection string for backwards compatibility.